docs-src/docs/guides/terraform_beyond_layer0.md

```diff
@@ -0,0 +1,519 @@
+# Deployment guide: Terraform beyond Layer0
+ In this example, we'll learn how you can use Terraform to create a Layer0 service as well as a persistent data store. The main goal of this example is to explore how you can combine Layer0 with other Terraform providers and best practices around how you can leverage functionality provided by Terraform.
+
+## Before you start
+To complete the procedures in this section, you must have the following installed and configured correctly:
+
+ * Layer0 v0.8.4 or later
+ * Terraform v0.9.0 or later
+ * Layer0 Terraform Provider
+
+If you have not already configured Layer0, see the [Layer0 installation guide](/setup/install). If you are running an older version of Layer0, see the [Layer0 upgrade instructions](/setup/upgrade#upgrading-older-versions-of-layer0).
+
+See the [Terraform installation guide](/reference/terraform-plugin#install) to install Terraform and the Layer0 Terraform Plugin.
+
+---
+
+## Deploy with Terraform
+Using Terraform, you will deploy a simple guestbook application, which is backed by AWS DynamoDB Table, for persistent storage. The terraform configuration file will use both the Layer0 and AWS Terraform providers, to deploy the guestbook application and provision a new DynamoDB Table.
+
+## Part 1: Clone the Layer0 examples repository
+Run this command to clone the `quintilesims/layer0-examples` repository. Once you have cloned the repository, navigate to `.terraform-beyond-layer0/example-1` folder for the rest of this example.
+
+`git clone https://github.com/quintilesims/layer0-examples.git`  
+
+## Part 2: Terraform Plan
+!!! Note
+	As we're using modules in our Terraform configuration we need to run `terraform get` before performing other terraform operations. Running get will download the modules to your local folder named `.terraform`. See here for more information on [terraform get](https://www.terraform.io/docs/commands/get.html).
+
+	`terraform get`
```

Comment:
Without running `terraform get` we would run into the error:

```
Failed to load root config module: Error loading modules: module guestbook: not found, may need to be downloaded using 'terraform get'
```

I have updated it to show an example output.